### PR TITLE
fix: enable mobile fade in drawer

### PR DESCRIPTION
**Pull request by Vikingjonsson**

### CODE REVIEW

**Comment by shmidt-i** [comment]
@mjmonline didn't we have it like this before? :)

File: src/Molecules/Drawer/Drawer.tsx [hunk 1]
```diff
@@ -127,7 +127,7 @@ export const Drawer = (React.forwardRef<HTMLDivElement, Props>(
                   <Icon.CrossThin size={CROSS_SIZE} title="Close this drawer" />
                 </CloseButton>
               </TitleWrapper>
-              <FadedScroll>{children}</FadedScroll>
+              <FadedScroll enableMobileFade>{children}</FadedScroll>
```
Comment (shmidt-i) [inline src/Molecules/Drawer/Drawer.tsx]: @mjmonline didn't we have it like this before? :)

Comment (mjmonline) [reply src/Molecules/Drawer/Drawer.tsx]: No, but this component should in my opinion not be baked in with the Drawer by default. The Modal component for example just takes children and there you can choose to have a FadedScroll or not

Comment (Vikingjonsson) [reply src/Molecules/Drawer/Drawer.tsx]: yes that sounds like a solution for this

**Pull request closed without merging**